frontend/workflows-lib/lib/components/template/SubmissionForm.tsx

```diff
@@ -73,6 +76,23 @@ const TemplateSubmissionForm: React.FC<TemplateSubmissionFormProps> = ({
         </Typography>
       )}
       <Divider />
+      {!!overriddenKeys?.length && (
+        <>
+          <Stack
+            direction={"row"}
+            alignItems={"flex-start"}
+            display="inline-flex"
+            spacing={theme.spacing(1)}
+          >
+            <Info fontSize="small" />
+            <Typography>
+              These parameters have been set via link:{" "}
```

Comment:
_These inputs have been set via merged parameters from [workflow name] and the url link:{" "}_

Something like that.

Reply:
I would remove the `Using initial parameters from [workflow name]` line at the top and merge it into a one-line message here